docs/dpdk-configuration.md

```diff
@@ -0,0 +1,77 @@
+<!--
+SPDX-License-Identifier: Apache-2.0
+Copyright 2022 Intel Corporation
+-->
+
+# DPDK Configuration
+
+If you are planning to use the UPF with DPDK, the following steps are required to properly configure the devices:
+
+- Clone `dpdk`
+```bash
+$ git clone https://github.com/DPDK/dpdk.git
```

Comment:
Are you sure about latest dpdk version ? I though Amar was saying that bess is tied up with specific version of BESS

Reply:
Yes, BESS works with dpdk-20.11.3. However, this step is just to bind the devices. It does not mean that BESS will use this (latest) version of DPDK

Reply:
hey Gabriel, its better to mention actual version of dpdk we prefer for binding as there is no guarantee of always having backward compatibility of dpdk latest vesrion, even for binding of devices and other operations.

Reply:
> its better to mention actual version of dpdk we prefer for binding as there is no guarantee of always having backward compatibility of dpdk latest vesrion, even for binding of devices and other operations.

I do not understand your comment. The device binding has nothing to do with BESS and that should be transparent to the DPDK version used for the binding